### ISSUE TITLE
<html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by DanielKim1** [NatVIII/rva.rip#54]
From a [Lighthouse](https://developer.chrome.com/docs/lighthouse/) accessibility audit:

> If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more about the lang attribute](https://dequeuniversity.com/rules/axe/4.8/html-has-lang).

I think you'll have to add an `htmlAttrs` property to the object being passed as the argument to `useHead`, and then add a `lang` property to `htmlAttrs` with the value `en-US` or something. See https://github.com/nuxt/nuxt/issues/388

Natalie! Don't do this one! Leave this one for a newcomer

**Comment by NatVIII** [comment]
I'm like a kid with their face on the glass cupping my eyes so I can look at this issue like the latest willy wonka candy but then I see `Natalie! Don't do this one! Leave this one for a newcomer` and I remember the mission

**Comment by NatVIII** [comment]
This happens atleast once a month
